crates/viewer/re_ui/src/list_item/property_content.rs

```diff
@@ -152,9 +152,14 @@ impl<'a> PropertyContent<'a> {
     /// Show a read-only boolean in the value column.
     #[inline]
     pub fn value_bool(self, mut b: bool) -> Self {
-        self.value_fn(move |ui: &mut Ui, _| {
-            ui.add_enabled_ui(false, |ui| ui.re_checkbox(&mut b, ""));
-        })
+        if true {
+            self.value_text(b.to_string())
+        } else {
+            // This is not readable, which is why it is disabled
```

Comment:
have you checked whether there are situations where value_bool would have looked better like that? Don't know if there are callsites where this is a standin for a disabled editable property 🤔

Reply:
I checked all calls of `value_bool`. There was only one.